src/test/resources/com/puppycrawl/tools/checkstyle/api/filetext/InputFileText.java

```diff
@@ -0,0 +1,18 @@
+package com.puppycrawl.tools.checkstyle.api.filetext;
+
+import java.util.regex.Pattern;
+
+public class InputFileText {
+    public static boolean isValidPasswd(String passwd) {
+        String reg = "^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[~#;:?" +
+                "/@&!\"'%*=¬.,-])(?=[^\\s]+$).{8,24}$";
+        return Pattern.matches(reg, passwd);
+    }
+}
+
+/**
+ * Test case for the "design for inheritance" check.
+ * @author Lars Kühne
```

Comment:
If we targeting in loading of `Kühne` ad `Khne`.
Please make check execution on this.

Reply:
@romani this test is already extending https://github.com/checkstyle/checkstyle/blob/e3d8c85805787a6af627ad5e7c133f67186a06c3/src/test/java/com/puppycrawl/tools/checkstyle/api/FileTextTest.java#L39

Reply:
Extension is not related.
Please use some Check in config to violate this line.

Reply:
@Kevin222004 Ping